vsphere-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1472 @@
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      topMemoryUtilizationHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top memory usage by ESXi hosts',
+          targets=[t.topMemoryUtilizationHosts],
+          description='The ESXi hosts with the highest memory utilization in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      topDiskAvgLatencyHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top avg disk latency by ESXi hosts',
+          targets=[t.topDiskAvgLatencyHosts],
+          description='The ESXi hosts with the highest average disk latency in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topPacketErrorRateHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top packet errors by ESXi hosts',
+          targets=[t.topPacketErrorRateHosts],
+          description='The ESXi hosts with the highest percentage of packet errors in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      hostCPUUsage:
+        commonlib.panels.generic.timeSeries.base.new(
+          'CPU usage',
+          targets=[t.hostCPUUsage],
+          description='The amount of CPU used by the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('rotmhz')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostCPUUtilization:
+        commonlib.panels.memory.timeSeries.usagePercent.new(
+          'CPU utilization',
+          targets=[t.hostCPUUtilization],
+          description='The CPU utilization percentage of the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percent')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostMemoryUsage:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Memory usage',
+          targets=[t.hostMemoryUsage],
+          description='The amount of memory used by the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('mbytes')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostMemoryUtilization:
+        commonlib.panels.memory.timeSeries.usagePercent.new(
+          'Memory utilization',
+          targets=[t.hostMemoryUtilization],
+          description='The memory utilization percentage of the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percent')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
```

Comment:
These four panels have the same squished legend issue, might benefit from list style or increased height:
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/68a6c38f-81b1-4c57-9982-70a2c5bbf5b3)